autotx/tests/agents/test_token_research.py

```diff
@@ -0,0 +1,55 @@
+from autotx.agents.TokenResearchAgent import TokenResearchAgent
+from crewai import Task
+
+agent = TokenResearchAgent()
+
+
+def test_price_change_information():
+    task = Task(
+        description="What's the 24 hours price change of Starknet (STRK)",
+        agent=agent,
+        expected_output="The price change of STRK in the last 24 hours",
+    )
+    response = agent.execute_task(task)
```

Comment:
NIT: I think the process of running a prompt against a single agent should be easier. Instead of creating a crewai Task instance, it'd be nice to simply run `agent.run(prompt)`. This ignores the `expected_output` property, but I'm not sure how that is used exactly, could either leave it blank or have the user pass it into the `agent.run(...)` method.

Reply:
i agree with you, but since we're working on top of crewai doing `agent.run` is not possible - we need to do things this way

Reply:
Couldn't we add our own method here? https://github.com/polywrap/AutoTx/blob/main/autotx/auto_tx_agent.py

Reply:
will do in upcoming PR!